libs/Zend/Session.php

```diff
@@ -479,14 +479,14 @@ public static function start($options = false)
                 restore_error_handler();
             }
 
-            if (!$startedCleanly || Zend_Session_Exception::$sessionStartError != null) {
+            if (!$startedCleanly || !empty(Zend_Session_Exception::$sessionStartError)) {
```

Comment:
ideally we would even only go into that logic if `!$startedCleanly` and not look at `Zend_Session_Exception::$sessionStartError` at all but I reckon we could still change that later.